README.md

```diff
@@ -391,7 +392,7 @@ They can be defined as environment variables or resource parameters (has priorit
 | azurerm_virtual_machine_disk, azurerm_virtual_machine_disks | `2017-03-30` | [azure_virtual_machine_disk](docs/resources/azure_virtual_machine_disk.md), [azure_virtual_machine_disks](docs/resources/azure_virtual_machine_disks.md) |
 | azurerm_virtual_network, azurerm_virtual_networks | `2018-02-01` | [azure_virtual_network](docs/resources/azure_virtual_network.md), [azure_virtual_networks](docs/resources/azure_virtual_networks.md) |
 | azurerm_webapp, azurerm_webapps | `2016-08-01` | [azure_webapp](docs/resources/azure_webapp.md), [azure_webapps](docs/resources/azure_webapps.md) |
-
+| azure_express_route_providers | `2021-07-29` | [azure_express_route_providers](docs/resources/azure_express_route_providers.md) |
```

Comment:
I don't think is needed .. this is for resources that were created initially with azurerm_* prefix so that users with older versions inspec could refer..

Reply:
but it is not blocking this PR from merging

Reply:
Ohh ok .Should I remove it or keep it for now ?